easybuild/easyconfigs/p/PGPLOT/PGPLOT-5.2.2-GCCcore-10.3.0.eb

```diff
@@ -34,15 +34,21 @@ dependencies = [
     ('X11', '20210518'),
 ]
 
-cmds_map = [('.*', './makemake . linux g77_gcc && make && make cpg')]
+cmds_map = [('.*', './makemake . linux g77_gcc && make && make shared && make cpg && make cpg-shared')]
 
 files_to_copy = [
     (['pgdemo*'], 'bin'),
-    (['libpgplot.a', 'libcpgplot.a'], 'lib'),
+    (['libpgplot.*', 'libcpgplot.*'], 'lib'),
     (['grfont.dat'], 'share'),
     (['cpgplot.h'], 'include'),
 ]
 
+postinstallcmds = [
+    # create symlinks
+    "cd %(installdir)s/lib && ln -s libcpgplot.so.5.2.2 libcpgplot.so.5 ; ln -s libcpgplot.so.5.2.2 libcpgplot.so && "
```

Comment:
I'm not sure if it makes sense, but couldn't this use the `SHLIB_EXT` template var instead of hardcodeing `.so`? See e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/b944e985eb489bcc88dec5da9364512d893fb989/easybuild/easyconfigs/n/nodejs/nodejs-14.17.6-GCCcore-11.2.0.eb

Reply:
I have tried to do this but I would need the `.so` `.so.5` and `.so.5.2.2` file extensions. That was the reason for my symlinking above. With what you suggested I only got one extension. Please let me know if there is a way of doing it using your suggestion. Thanks.

Reply:
fixed in [c074f0b](https://github.com/easybuilders/easybuild-easyconfigs/pull/15496/commits/c074f0bb3110c27d9969c3d0b19dde3eca868bd4)